love/src/components/Layout/Layout.jsx

```diff
@@ -271,6 +275,85 @@ class Layout extends Component {
+          </svg>
+        );
+      case 'Summit':
+        return (
+          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 17.12 18.46">
+            <path className={style} d="m1.15,17.54l1.61-4.14c.23-.59.94-.75,1.38-.32h0s2.11-6.49,2.11-6.49c.18-.55.51-1.04.95-1.4h0c1.38-1.12,3.37-.68,4.21.93l.76,1.47h0c.53-.19,1.1.09,1.32.64l3.59,9.3c.17.44-.13.92-.58.92H1.73c-.45,0-.75-.48-.58-.92Z"/>
+            <path className={style} d="m4.02,2.95l-1.2.35c-.15.06-.15.26,0,.32l1.2.35s.08.05.1.1l.35,1.2c.06.15.26.15.32,0l.35-1.2s.05-.08.1-.1l1.2-.35c.15-.06.15-.26,0-.32l-1.2-.35s-.08-.05-.1-.1l-.35-1.2c-.06-.15-.26-.15-.32,0l-.35,1.2s-.05.08-.1.1Z"/>
+            <path className={style} d="m10.21,1.33l-.76.22c-.09.04-.09.17,0,.2l.76.22s.05.03.06.06l.22.76c.04.09.17.09.2,0l.22-.76s.03-.05.06-.06l.76-.22c.09-.04.09-.17,0-.2l-.76-.22s-.05-.03-.06-.06l-.22-.76c-.04-.09-.17-.09-.2,0l-.22.76s-.03.05-.06.06Z"/>
+            <circle className={style} cx="2.34" cy="1.05" r="1.05"/>
+            <circle className={style} cx="13.83" cy="3.79" r="1.05"/>
+            <circle className={style} cx="7.78" cy=".56" r=".52"/>
+            <circle className={style} cx=".44" cy="3.43" r=".44"/>
+            <circle className={style} cx="15.99" cy="1.74" r=".44"/>
+          </svg>
+        );
+      case 'LSST':
+        return (
+          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 18.49 16.74">
+            <path className={style} d="m4.18.07L1.11,1.78c-.68.38-1.11,1.14-1.11,1.96v13h5.02V.62c0-.46-.45-.76-.83-.55Z"/>
+            <path className={style} d="m17.72,6.04H5.86v10.71h2.69v-2.28h7.26v2.28h2.69V6.81c0-.43-.35-.77-.77-.77Zm-8.86,4.2h-1.31v-2.38h1.31v2.38Zm5.91,0h-5.18v-2.38h5.18v2.38Zm2.04,0h-1.31v-2.38h1.31v2.38Z"/>
+          </svg>
+        );
+      default:
+        return (
+          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 14 19">
+            <path className={style} d="m7,0C3.13,0,0,3.13,0,7s7,12,7,12c0,0,7-8.13,7-12S10.87,0,7,0Zm0,12c-2.76,0-5-2.24-5-5S4.24,2,7,2s5,2.24,5,5-2.24,5-5,5Z"/>
+            <path className={style} d="m7,9.52c-.28,0-.5-.22-.5-.5v-1.68c0-.28.22-.5.5-.5.83,0,1.5-.67,1.5-1.5s-.67-1.5-1.5-1.5-1.5.67-1.5,1.5c0,.28-.22.5-.5.5s-.5-.22-.5-.5c0-1.38,1.12-2.5,2.5-2.5s2.5,1.12,2.5,2.5c0,1.21-.86,2.22-2,2.45v1.23c0,.28-.22.5-.5.5Z"/>
+            <circle className={style} cx="7" cy="10.6" r=".5"/>
+          </svg>
+        );
+    }
+
+
+    return(
+      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 19.84 20">
+        <path
+          className={style}
+          d="m17.84,6.35c-1.1,0-2,.9-2,2v3.8c0,.44-.36.81-.81.81h-1.11V4c0-2.21-1.79-4-4-4s-4,1.79-4,4v4.95h-1.11c-.44,0-.81-.36-.81-.81v-3.8c0-1.1-.9-2-2-2S0,3.24,0,4.35v3.8c0,2.65,2.16,4.81,4.81,4.81h1.11v6.05c0,.55.45,1,1,1h6c.55,0,1-.45,1-1v-2.05h1.11c2.65,0,4.81-2.16,4.81-4.81v-3.8c0-1.1-.9-2-2-2Z"/>
+      </svg>
```

Comment:
I would move each of these `svg` elements to a file, similar as we do with Icons.